<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-18T18:07:42 IST" name="Default suite" finished-at="2022-11-18T18:08:00 IST" duration-ms="18927">
    <groups>
    </groups>
    <test started-at="2022-11-18T18:07:42 IST" name="Default test" finished-at="2022-11-18T18:08:00 IST" duration-ms="18927">
      <class name="Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass">
        <test-method is-config="true" signature="OpenBrowser()[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a]" started-at="2022-11-18T18:07:42 IST" name="OpenBrowser" finished-at="2022-11-18T18:07:50 IST" duration-ms="8148" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenBrowser -->
        <test-method is-config="true" signature="Login()[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a]" started-at="2022-11-18T18:07:50 IST" name="Login" finished-at="2022-11-18T18:07:56 IST" duration-ms="6705" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method signature="VerifyUserId()[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a]" started-at="2022-11-18T18:07:56 IST" name="VerifyUserId" finished-at="2022-11-18T18:08:00 IST" duration-ms="3241" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Failed expected [Harshada P.4] but found [Harshada P.],
Failed expected [Upstox Pro555] but found [Upstox Pro]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Failed expected [Harshada P.4] but found [Harshada P.],
Failed expected [Upstox Pro555] but found [Upstox Pro]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass.VerifyUserId(UpstoxTestClass.java:82)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUserId -->
        <test-method is-config="true" signature="PostConditions(org.testng.ITestResult)[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a]" started-at="2022-11-18T18:08:00 IST" name="PostConditions" finished-at="2022-11-18T18:08:00 IST" duration-ms="206" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyUserId status=FAILURE method=UpstoxTestClass.VerifyUserId()[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostConditions -->
        <test-method is-config="true" signature="CloseBroswer()[pri:0, instance:Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass@5ba3f27a]" started-at="2022-11-18T18:08:00 IST" name="CloseBroswer" finished-at="2022-11-18T18:08:00 IST" duration-ms="554" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBroswer -->
      </class> <!-- Pom_DDF_testNG_BaseClass_UtilityClass.UpstoxTestClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
